content/docs/nav.yml

```diff
@@ -1,85 +1,85 @@
-- title: Installation
+- title: Kurulum
   items:
     - id: getting-started
-      title: Getting Started
+      title: Başlangıç
     - id: add-react-to-a-website
-      title: Add React to a Website
+      title: Bir Web Sitesine React Eklemek
     - id: create-a-new-react-app
-      title: Create a New React App
+      title: Yeni bir React Uygulaması Başlatmak
     - id: cdn-links
-      title: CDN Links
-- title: Main Concepts
+      title: CDN Bağlantıları
+- title: Temel Kavramlar
   isOrdered: true
   items:
     - id: hello-world
-      title: Hello World
+      title: Merhaba Dünya
     - id: introducing-jsx
-      title: Introducing JSX
+      title: JSX'e Giriş
     - id: rendering-elements
-      title: Rendering Elements
+      title: Elementlerin Render Edilmesi
     - id: components-and-props
-      title: Components and Props
+      title: Bileşenler ve Prop'lar
     - id: state-and-lifecycle
-      title: State and Lifecycle
+      title: State ve Yaşam Döngüsü
     - id: handling-events
-      title: Handling Events
+      title: Olay Yönetimi
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: Koşullu Renderlama
     - id: lists-and-keys
-      title: Lists and Keys
+      title: Listeler ve Anahtarlar
     - id: forms
-      title: Forms
+      title: Formlar
     - id: lifting-state-up
-      title: Lifting State Up
+      title: State'i Yukarı Taşıma
     - id: composition-vs-inheritance
-      title: Composition vs Inheritance
+      title: Bileşim vs Kalıtım
     - id: thinking-in-react
-      title: Thinking In React
-- title: Advanced Guides
+      title: React'te Düşünmek
+- title: Gelişmiş Kılavuz
   items:
     - id: accessibility
-      title: Accessibility
+      title: Erişebilirlik
     - id: code-splitting
-      title: Code-Splitting
+      title: Kod Bölümleme
     - id: context
-      title: Context
+      title: Bağlam (Context)
```

Comment:
proposal expecting for `Context`

Reply:
Ben de `Hooks` ve `Context` kelimelerinin orijinalleri gibi kalmasını tercih ediyorum. Zaten `Context`'i `Bağlam` olarak çevirdiğimizde daha da anlamsizlasiyor